kedro/framework/cli/starters.py

```diff
@@ -1005,8 +1005,7 @@ def __str__(self) -> str:
 
     def validate(self, user_input: str) -> None:
         """Validate a given prompt value against the regex validator"""
-
-        if self.regexp and not re.match(self.regexp, user_input.lower()):
+        if self.regexp and not re.match(self.regexp, user_input):
             message = f"'{user_input}' is an invalid value for {(self.title).lower()}."
```

Comment:
Since the `.lower()` call is removed above, I think it should be removed from the error message as well to avoid confusion about what input was given.

Reply:
The `lower()` here actually transforms the title of the corresponding prompt (e.g. "Project Name", "Project Tools") and not the user input. I would rather keep it to stay it consistent with the other logs here (e.g. [L202](https://github.com/kedro-org/kedro/blob/910fa7166726446baa69863bd9418f97d643ba63/kedro/framework/cli/starters.py#L202), [L912)](https://github.com/kedro-org/kedro/blob/910fa7166726446baa69863bd9418f97d643ba63/kedro/framework/cli/starters.py#L912).

What do you think?

Reply:
Ah yes I see! Let's leave it then 🙂